modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/java/com/liferay/osb/faro/web/internal/controller/main/ChannelController.java

```diff
@@ -129,7 +129,7 @@ public FaroChannelDisplay create(
 
 		Channel channel = new Channel();
 
-		channel.setName(name);
+		channel.setName(HtmlUtil.escapeAttribute(name));
```

Comment:
@marcosapmf , I don't think we should transform data before saving it to the database.

Reply:
@marcellustavares, the motivation of this is that we sanitize the keywords for searching the channel by name to prevent SQL Injection, but when we do that, the keywords does not match to the value we save.
Escaping the name, not only fixes this issue, but also prevents other kind of attacks, like Code Injection.

Reply:
@marcellustavares, I took a look at the method we use to make the query and it is using `PreparedStatement`, in light of this, we don't need to worry about escaping the variable value :smile: